src/container.cpp

```diff
@@ -655,7 +653,7 @@ uint32_t Container::getItemTypeCount(uint16_t itemId, int32_t subType /* = -1*/)
 
 std::map<uint32_t, uint32_t>& Container::getAllItemTypeCount(std::map<uint32_t, uint32_t>& countMap) const
 {
-	for (Item* item : itemlist) {
+	for (auto item : itemList) {
```

Comment:
Could this be `const auto`
```suggestion
	for (const auto& item : itemList) {
```